src/gui/SetupDialog.cpp

```diff
@@ -1044,6 +1063,8 @@ void SetupDialog::accept()
 	ConfigManager::inst()->setValue( "ui", "disableautoquit",
 					QString::number( m_disableAutoQuit ) );
 	ConfigManager::inst()->setValue( "app", "language", m_lang );
+	ConfigManager::inst()->setValue( "ui", "vstembedmethod",
+					m_vstEmbedComboBox->currentData().toString() );
```

Comment:
`QComboBox::currentData()` was introduced in Qt 5.2 and breaks Qt4 compatibility. Are there any alternatives? Or just drop Qt4 support now?

Reply:
I found an alternative: `m_vstEmbedComboBox->itemData(m_vstEmbedComboBox->currentIndex()).toString()`.

Reply:
I suggest we drop Qt4 support with this PR. It's too much work to maintain both versions, and the VST issue is the last remaining deal-breaker with Qt5.